suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -385,6 +385,11 @@ export const selectAccountBalance = createMemoizedSelector(
     account => account?.balance ?? null,
 );
 
+export const selectAccountFormattedBalance = createMemoizedSelector(
```

Comment:
Maybe `selectAccountCryptoBalanceWithStaking` would be better or staking is not expected here?

Reply:
I believe that is not expected there.